src/HealthGPS/static_linear_model.cpp

```diff
@@ -112,7 +112,12 @@ void StaticLinearModel::initialise_factors(Person &person, Random &random) const
         double expected = get_risk_factor_expected().at(person.gender, names_[i]).at(person.age);
         double factor = linear[i] + residual * stddev_[i];
         factor = expected * inverse_box_cox(factor, lambda_[i]);
-        person.risk_factors[names_[i]] = factor;
+
+        // Clamp risk factor to range.
+        double factor_clamped = ranges_[i].clamp(factor);
+
+        // Save risk factor.
+        person.risk_factors[names_[i]] = factor_clamped;
```

Comment:
Can this be simplified as..

```suggestion
        // Save risk factor.
        person.risk_factors[names_[i]] = ranges_[i].clamp(factor);
```
?